<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>881</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableWidget" name="data_table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>511</width>
       <height>461</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(186, 189, 182);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <column>
      <property name="text">
       <string>Year</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Agriculture</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Industrial</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Services</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Domestic</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="barchart">
     <property name="geometry">
      <rect>
       <x>558</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 138, 133);</string>
     </property>
     <property name="text">
      <string>Bar Chart</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Piechart">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>20</y>
       <width>89</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 138, 133);</string>
     </property>
     <property name="text">
      <string>Pie Chart</string>
     </property>
    </widget>
    <widget class="QPushButton" name="extrapolate">
     <property name="geometry">
      <rect>
       <x>558</x>
       <y>80</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 138, 133);</string>
     </property>
     <property name="text">
      <string>extrapolation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="compBar">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>80</y>
       <width>89</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 138, 133);</string>
     </property>
     <property name="text">
      <string>Component</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>511</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 138, 133);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>221</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(206, 92, 0);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Year.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showyear">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);</string>
      </property>
      <property name="text">
       <string>Show</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rawdata">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>240</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(17, 65, 47);</string>
      </property>
      <property name="text">
       <string>Show Raw Data</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>190</y>
       <width>291</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(211, 215, 207);</string>
     </property>
     <property name="title">
      <string>T Test</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Var1</string>
      </property>
     </widget>
     <widget class="QComboBox" name="var1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>86</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Var2</string>
      </property>
     </widget>
     <widget class="QComboBox" name="var2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>86</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="showt">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Click Here</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="tOutput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>261</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(211, 215, 207);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_5</zorder>
    <zorder>data_table</zorder>
    <zorder>barchart</zorder>
    <zorder>Piechart</zorder>
    <zorder>extrapolate</zorder>
    <zorder>compBar</zorder>
    <zorder>groupBox_6</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>891</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border-radius:10px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>801</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>5</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(196, 160, 0);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>42</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(211, 215, 207);
background-color: rgb(196, 160, 0);</string>
      </property>
      <property name="text">
       <string>DSS</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Helping Analyze Data</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>801</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(164, 0, 0);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="SelectFile">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>361</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(46, 52, 54);
border-radius:20px;</string>
      </property>
      <property name="text">
       <string>Select File</string>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_2</zorder>
    <zorder>groupBox</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
